DP3TApp/Logic/Networking/Base/UBPinnedCertificatesTrustEvaluator.swift

```diff
@@ -185,10 +185,10 @@ extension SecTrust {
     ///                            `Error`.
     /// - Throws:                  The `Error` produced by the `errorProducer` closure.
     func validate() throws {
-        var result = SecTrustResultType.invalid
-        let status = SecTrustEvaluate(self, &result)
+        var error: CFError?
+        let status = SecTrustEvaluateWithError(self, &error)
```

Comment:
I think we should use `SecTrustEvaluateAsyncWithError` instead, as this method is blocking. Or otherwise schedule it on another thread.

Reply:
Since URLSession already calls us in background it should be fine though